docs/deploy-full-rp-service-in-dev.md

```diff
@@ -29,11 +29,42 @@
     make dev-config.yaml
     ```
 
-
 1. Run `make deploy`
-    > __NOTE:__ This will fail on the first attempt to run due to certificate and container mirroring requirements.
+    > __NOTE:__ This will fail on the first attempt to run due to certificate or AKS requirements.
+
+    > __NOTE:__ If the deployment fails with `InvalidResourceReference` due to the RP Network Security Groups not found, delete the "gateway-production-predeploy" deployment in the gateway resource group, and re-run `make deploy`.
+
+    > __NOTE:__ If the deployment fails with `A vault with the same name already exists in deleted state`, then you will need to recover the deleted keyvaults from a previous deploy using: `az keyvault recover --name <KEYVAULT_NAME>` for each keyvault, and re-run.
+
+1. Deploy AKS by running these commands from the ARO-RP root directory:
+    ```bash
+    source ./hack/devtools/deploy-shared-env.sh
+    deploy_aks_dev
+    ```
+    > __NOTE:__ If the AKS deployment fails with missing RP VNETs, delete the "gateway-production-predeploy" deployment in the gateway resource group, and re-run `make deploy` and then re-run `deploy_aks_dev`.
 
-    > __NOTE:__ If the deployment fails with `InvalidResourceReference` due to the RP Network Security Groups not found, delete the gateway predeploy deployment, and re-run.
+1. Install Hive into AKS
+    1. Deploy a VPN Gateway and in order to be able to connect to AKS from your local machine:
+        ```bash
+        source ./hack/devtools/deploy-shared-env.sh
+        deploy_vpn_for_dedicated_rp
+        ```
+
+    1. Download the VPN config. Please note that this action will _**OVER WRITE**_ the `secrets/vpn-$LOCATION.ovpn` on your local machine. **DO NOT** run `make secrets-update` after doing this, as you will overwrite existing config, until such time as you have run `make secrets` to get the config restored.
+        ```bash
+        vpn_configuration
+        ```
+
+    1. Connect to the Dev VPN in a new terminal:
+        ```bash
+        sudo openvpn secrets/vpn-$LOCATION.ovpn
+        ```
+
```

Comment:
After connecting to the vpn and before installing hive we need to run `make aks.kubeconfig`

Reply:
Added, thanks.